uikit/src/main/java/com/midtrans/sdk/uikit/activities/CreditDebitCardFlowActivity.java

```diff
@@ -748,4 +765,92 @@ private void bindSavedCards(ArrayList<SaveCardRequest> cards) {
     }
 
 
+    public void getBankBins() {
```

Comment:
Instead of making this as method in `CreditDebitCardFlowActivity`, can you make this as utils method?